<?xml version="1.0"?>

<launch>
    <!-- Load the simulation parameters -->
    <rosparam command="load" file="$(find pd_controller)/config/parameters.yaml"/>

    <!-- To retrieve OptiTrack data -->
    <arg name="server" default="172.26.213.1"/>
    <node pkg="vrpn_client_ros" type="vrpn_client_node" name="vrpn_client_node" output="screen">
    <rosparam subst_value="true">
        server: $(arg server)
        port: 3883
        frame_id: 1
        broadcast_tf: true
        # Must either specify refresh frequency > 0.0, or a list of trackers to create
        refresh_tracker_frequency: 1.0
        #trackers:
        #- FirstTracker
        #- SecondTracker
    </rosparam>
    </node>
    
    <node name="main_node" pkg="pd_controller" type="steering_car_main_node.py" output="screen">
    </node>

    <node name="trajectory_node" pkg="trajectory" type="trajectory_node.py" output="screen">
    </node>
    
    
</launch>
